.github/actions/spell-check/patterns.txt

```diff
@@ -60,3 +60,6 @@ C0FFEE
 # DoT: DNS over TLS is often combined with other words (images, links, ...), same for DoH
 DoT
 DoH
+
+# Contributor with a non-ascii character in their name
+Hoffst[^[:ascii:]]tte
```

Comment:
```suggestion
Hoffst..tte
```

Reply:
If you wanted to be stricter, this would work:
```suggestion
Hoffst[^[:ascii:]]+tte
```

Reply:
ah, right, it counts as two characters!